src/app/signup/member/page.tsx

```diff
@@ -0,0 +1,388 @@
+  [UserFormKey.DISTRICT]: string;
+  [UserFormKey.GOALS]: string[];
+  [UserFormKey.CATEGORY]: string[];
+}
+
+interface StepProps {
+  next?: () => void;
+  prev?: () => void;
+}
+const { Title, Text } = Typography;
+
+const Step1 = ({ next }: StepProps) => {
+  const {
+    control,
+    formState: { errors, isValid },
+    watch,
+  } = useFormContext<IMemberFormData>();
+  const watchedName = watch(UserFormKey.NICKNAME);
+  const watchedPhone = watch(UserFormKey.CELLPHONE);
+
+  return (
+    <>
+      <Controller
+        name={UserFormKey.NICKNAME}
+        control={control}
+        render={({ field, fieldState }) => (
+          <>
+            <Form.Item
+              style={{ width: "100%" }}
+              validateStatus={errors[UserFormKey.NICKNAME] ? "error" : ""}
+              help={errors[UserFormKey.NICKNAME] ? errors[UserFormKey.NICKNAME]?.message : ""}
+            >
+              <Typography>닉네임</Typography>
+              <Input
+                {...field}
+                placeholder="특수 문자 제외(2~5글자)"
+                id="nickname-input"
+                value={field.value}
+                onChange={field.onChange}
+                status={fieldState.error && "error"}
+              />
+            </Form.Item>
+          </>
+        )}
+      />
+      <Controller
+        name={UserFormKey.CELLPHONE}
+        control={control}
+        render={({ field, fieldState }) => {
+          return (
+            <Form.Item
+              validateStatus={errors[UserFormKey.CELLPHONE] ? "error" : ""}
+              help={errors[UserFormKey.CELLPHONE] ? errors[UserFormKey.CELLPHONE]?.message : ""}
+            >
+              <Typography>핸드폰 번호</Typography>
+              <Input
+                {...field}
+                placeholder="핸드폰 번호 입력"
+                id="nickname-input"
+                value={field.value}
+                onChange={field.onChange}
+                status={fieldState.error && "error"}
+                color={fieldState.error && "error"}
+              />
+            </Form.Item>
+          );
+        }}
+      ></Controller>
+      <Form.Item>
+        <Button
+          disabled={
+            !watchedName.length ||
+            !watchedPhone.length ||
+            Boolean(errors[UserFormKey.NICKNAME]) ||
+            Boolean(errors[UserFormKey.CELLPHONE])
+          }
+          style={{ width: "100%" }}
+          type="primary"
+          size="large"
+          onClick={next}
+          htmlType="submit"
+        >
+          다음
+        </Button>
+      </Form.Item>
+    </>
+  );
+};
+
+const Step2 = ({ next }: StepProps) => {
+  const {
+    control,
+    formState: { errors },
+    watch,
+  } = useFormContext<IMemberFormData>();
+  const [districtOptions, setDistrictOptions] = useState<OptionValue[] | null>([]);
+  const watchedAge = watch(UserFormKey.AGE);
+  const watchedCity = watch(UserFormKey.CITY);
+  const watchedDistrict = watch(UserFormKey.DISTRICT);
```

Comment:
```suggestion
const [age, city, district] = watch([UserFormKey.AGE, UserFormKey.CITY, UserFormKey.DISTRICT])
```

이런식으로도 사용할 수 있을거야 !

그리고 useWatch() 라는 훅도 있는데 그거 사용하면 좋을듯 👍
watch 랑 useWatch 랑 성능차이가 있어서 - [참고](https://toby2009.tistory.com/50)